docs/Actions.md

```diff
@@ -103,6 +103,19 @@ const { data: user, isLoading, error } = useGetOne(
 );
 ```
 
+## `meta` Parameter
+
+All Data Provider methods accept a `meta` parameter. React-admin doesn't set this parameter by default in its queries, but it's a good way to pass special arguments or metadata to an API call.
+
+```jsx
```

Comment:
Why? We're using jsx elsewhere. 

I know there is no jsx here, but the js and jsx highlight colors aren't the same. If we don't use jsx everywhere, this will look odd.

Reply:
Got it